cmd/noms/splore/.flowconfig

```diff
@@ -0,0 +1,58 @@
+[ignore]
+; We fork some components by platform
```

Comment:
I know this is just a copy from our React Native app but this is not really relevant.